formatTest/typeCheckedTests/expected_output/oo.re

```diff
@@ -14,12 +14,18 @@ class virtual stack ('a) (init) = {
       Some(hd);
     | [] => None
     };
-  pub push = hd => v = [hd, ...v];
-  initializer (
-    print_string("initializing object")
-  );
-  pub explicitOverrideTest = a => a + 1;
-  pri explicitOverrideTest2 = a => a + 1;
+  pub push = hd => {
+    v = [hd, ...v];
+  };
+  initializer {
```

Comment:
What's the rational behind printing braces here, while the original had parens?

Reply:
@anmonteiro ?

Reply:
The original has braces: https://github.com/anmonteiro/reason/blob/373ed5f64d2f2a389e5a507270178dc5b1bf7efa/formatTest/typeCheckedTests/input/oo.re#L22